docs/index.md

```diff
@@ -15,13 +15,14 @@ terraform {
   required_providers {
     mondoo = {
       source  = "mondoohq/mondoo"
-      version = ">= 0.4.0"
+      version = ">= 0.19"
```

Comment:
This is intentional since we will release version `0.19.0` and I want all the examples to point to that as a min_version